js/game.js

```diff
@@ -0,0 +1,253 @@
+import {Player} from './players.js';
+import {Board} from './board.js';
+
+function returnAccumulatedScores(data) {
+
+  let dataToAccumulate = [];
+
+  do {
+
+    let elementToCompare = data.shift();
+    elementToCompare.apperances = 0;
+    for (let i = 0; i < data.length; i++) {

```

Comment:
jak wyciągniesz `data.length` do zmiennej przed pętlę to ja zoptymalizujesz. `JS` co każdy obrót pętli musi sprawdzać `data.length`, a tak będzie miał to w zmiennej więc nie będzie musiał wyszukiwać tych danych

Reply:
Ok